bin/start.sh

```diff
@@ -73,13 +77,10 @@ echo "Starting tail of gitlab-shell log file to output all logs to stdout"
 (tail -f /home/git/gitlab-shell/gitlab-shell.log ; echo "gitlab-shell logging stopped unexpectedly") &
 
 ### Setup permissions
-chown -R git:git /home/git
-chown root:root $GIT_REPO_MOUNT
 chmod -R o-rwx $GIT_REPO_ROOT
 chmod 700 $GIT_REPO_ROOT
 
 ## Enable non root ssh by removing nologin
-rm -f /run/nologin
```

Comment:
This one is related with rhel/fedora package bug. See: https://forum.opnsense.org/index.php?topic=916.0 for this workaround. I think we should still have it no matter what user would be used.